docs/05_best-practices/04_data-design-and-migration.md

```diff
@@ -2,9 +2,9 @@
 content_title: Data design and migration
 ---
 
-EOSIO based blockchains allow developers to easily update their smart contract code. However, a few things need to be considered when it comes to data update and/or migration. The main structure for storing data in EOSIO based blockchains is the multi index table. Once a multi index table has been created with a first version of a smart contract, it has some limitations when it comes to changing its structure. Below you will find a few possible approaches which you can consider when you design your smart contract data and its migration. 
+EOSIO based blockchains allow developers to easily update their smart contract code. However, a few things need to be considered when it comes to data updates and/or migration. The multi-index table API is one of the mechanisms (Key-Value API being the other) for storing and updating blockchain state. The multi-index table API creates and uses data structures in RAM. Once created and deployed on the blockchian, changing these structures, i.e. adding new fields, it has some limitations. Below you will find a few possible approaches which you can consider when you design your smart contract data and its migration. 
```

Comment:
Once created and deployed on the blockchian, changing these structures, i.e. adding new fields, it has some limitations.
=> maybe =>
Once created and deployed on the blockchAIn there are limitations to updating these structures.

Reply:
Below you will find a few possible approaches which you can consider when you design your smart contract data and its migration.
=> maybe =>
Below you will find two approaches to your smart contract data design, updates to this design, and for data migration.

Reply:
Ovi - I think you are using too many words - it make it more complicated to read

Reply:
> Ovi - I think you are using too many words - it make it more complicated to read
these files were written well before we put in place our conventions.
I updated these files (outside of the how tos) just to replace multi index with multi-index.
now that you provide feedback on these files as well is welcomed but it is outside the scope of this PR.